app/controllers/remote_clinic/friends_controller.rb

```diff
@@ -1,18 +1,30 @@
 class RemoteClinic::FriendsController < ApplicationController
   before_action :authenticate_user!
-  before_action :require_access_to_friend, only: [:show]
+  before_action :require_access_to_friend, only: [:show, :destroy]
 
   def index
     @friends = current_user.remote_clinic_friends.with_active_applications
   end
 
   def show
-    @friend = Friend.find_by(id: params[:id])
+    if !release
```

Comment:
i wanted to put the edgier case first, but it kinda reads... a tiny bit weird potentially. can reverse.